develop.html.mako

```diff
@@ -27,22 +27,40 @@ on getting started.
 <a name="developers"></a>
 <h2>Development Community</h2>
 
-<p>Where do you find the developers ?   Three places:</p>
+<p>Where do you find the developers ?  Four places:</p>
 
 <ul>
-    <li>The <a href="/support.html#mailinglist">main
-    mailing list</a> is the most common - core devs assist users with issues of all kinds.</li>
-    <li>Developer discussion takes place
-    on the <a href="http://groups.google.com/group/sqlalchemy-devel">development mailing list</a>,
-    also hosted on Google groups.  Anyone can join this
-    list, though we ask that discussion be limited to architectural and developmental issues only.
-    </li>
-    <li>Some development discussion occurs on the
-    <code>#sqlalchemy-devel</code> IRC channel on Freenode.
-    This channel is <strong>only</strong> for users that are interested in contributing code,
-    tests, documentation, or other developmental resources, and no support issues will be
-    addressed here.   IRC-based support is available on the
-    <a href="/support.html#irc">main IRC channel</a>.</li>
+    <li>The <a href="/support.html#mailinglist">main mailing list</a> is the most 
+        common - core devs assist users with issues of all kinds.
+        </li>
+    <li>Developer discussion takes place on the 
+        <a href="http://groups.google.com/group/sqlalchemy-devel">development
+        mailing list</a>, also hosted on Google groups.  Anyone can join this list, 
+        though we ask that discussion be limited to architectural and developmental 
+        issues only.
+        </li>
+    <li>Some development discussion occurs on the <code>#sqlalchemy-devel</code>
```

Comment:
I think this should talk about gitter as where this discussion occurs right now.  the paragraph can also mention the IRC channel, but as "there is also a #sqlalchemy-devel irc channel, however currentrly the gitter channel is where most develpoment discussion is occurring".